src/plugins/discover/public/application/main/utils/use_discover_state.ts

```diff
@@ -240,7 +228,6 @@ export function useDiscoverState({
     onChangeIndexPattern,
     onUpdateQuery,
     searchSource,
-    setState,
```

Comment:
this is a cleanup, right? so we could have removed `setState` before this PR, right?